prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -499,14 +499,14 @@ class ReportFunction : Logging {
                 workflowEngine.dispatchReport(event, batchReport, actionHistory, receiver, txn, context)
                 loggerMsg = "Queue: ${event.toQueueMessage()}"
             }
-            receiver.format == Report.Format.HL7 -> {
+            receiver.format.isSingleItemFormat -> {
```

Comment:
Single item formats are a concept in other parts of the code. This change allows new single-item formats to be defined without changing this piece of code.